Hypermedia.AspNetCore.Siren/Environments/ScopedBuildApplier.cs

```diff
@@ -4,14 +4,15 @@
     using System;
     using System.Threading.Tasks;
 
-    internal class ScopedBuildApplier<TBuilder, TBuilt> : IScopedBuildApplier<TBuilder, TBuilt> where TBuilder : class, IBuilder<TBuilt>
+    internal class ScopedBuildApplier<TBuilder, TBuilt> : IScopedBuildApplier<TBuilder, TBuilt>
+        where TBuilder : class, IBuilder<TBuilt>
         where TBuilt : class
```

Comment:
I was thinking about something like this:   
internal class ScopedBuildApplier<TBuilder, TBuilt> 
    : IScopedBuildApplier<TBuilder, TBuilt> where TBuilder : class, 
    IBuilder<TBuilt> where TBuilt : class

But this is only an opinion. :)